man/sysusers.d.xml

```diff
@@ -119,6 +119,15 @@ r     -        500-900
           bearing the same name unless the ID field specifies it. The account will be
           created disabled, so that logins are not allowed.</para>
 
+          <para>The line type <varname>u</varname> may be suffixed with an exclamation mark
+          (<literal>u!</literal>) in which case a fully locked account is created. This is typically
+          recommended, unless the specified user shall be accessible for login (which is the exception for
+          system users). Note the difference to lines without the exclamation mark: they result in an
```

Comment:
Something's off with the tenses here. Maybe "This is recommended, since logins should not typically be allowed for system users.".  Also, `!` is called a flag later on, so let's not say "type".

The whole text:
```
`Type <varname>u</varname> may be suffixed with an exclamation mark (<literal>u!</literal>)
to create a fully locked account.
This is recommended, since logins should not typically be allowed for system users.
In both cases an invalid password is set.
For `u!`, the account is also locked, which makes a difference for non-password forms of authentication, 
such as SSH or similar.
```
?